app/controllers/repositories_controller.rb

```diff
@@ -0,0 +1,74 @@
+class RepositoriesController < ApplicationController
+  after_action :mark_everything_as_read, only: [:index, :show]
+  before_action :print_read_status, only: :index
+
+  @client = Octokit::Client.new(:access_token => ENV['PERSONAL_ACCESS_TOKEN'])
```

Comment:
So ivars declared in the "root" of a class is actually going to define them on the singleton rather than on the controller. The normal way to write something like this would be:
```
private

def client
  @client ||= Octokit::Client.new(...)
  end
```

This will set `@client` only if it's `nil` - basically it's a lazy initializer.